package.json

```diff
@@ -55,24 +55,23 @@
   "devDependencies": {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31515&commitId=496afae1887e9dafbb3f53d07ee88a83a78c72aa&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Removed|
|Card Converged.appearance - RTL.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>